.mailmap

```diff
@@ -284,3 +284,9 @@ Jainul Vaghasia <jainulvaghasia@gmail.com> Jainul Vaghasia <jainulvaghasia@D-10-
 Arif Ahmed <arif.ahmed.5.10.1995@gmail.com> ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
 Pranjal Tale <pranjaltale16@gmail.com> = <pranjaltale16@gmail.com>
 jtoka <jason.tokayer@gmail.com> jtoka <jason.tokayer@capitalone.com>
+Theodore Han <theodorehan@hotmail.com> LAPTOP-4VOQQ6GI\han <theodorehan373@gmail.com>
+Theodore Han <theodorehan@hotmail.com> Theodore Han <tqhan317@gmail.com>
```

Comment:
This is .mailmap. Every unique name/email in the git history has to have a line here.